/path_manager:
  ros__parameters:
    num_drones: 3
    drone_0:
      drone_id: 0
      start_point_x: -2.0
      start_point_y: 0.0
      start_point_z: 0.5
      end_point_x: 11.0
      end_point_y: 13.0
      end_point_z: 0.5
    drone_1:
      drone_id: 1
      start_point_x: 0.0
      start_point_y: 0.0
      start_point_z: 0.5
      end_point_x: 12.0
      end_point_y: 13.0
      end_point_z: 0.5
    drone_2:
      drone_id: 2
      start_point_x: 2.0
      start_point_y: 0.0
      start_point_z: 0.5
      end_point_x: 12.0
      end_point_y: 12.0
      end_point_z: 0.5
